app/config/ezplatform.yml

```
@@ -25,9 +25,10 @@ ezpublish:
         match:
             URIElement: 1
 
-    # System settings, grouped by siteaccess and/or siteaccess group
+    # System settings, grouped by default (global), siteaccess and/or siteaccess group
+    # TIP: For multi site install organize shared config in groups, with single site we can set shared config in default
```

```suggestion
    # TIP: For multisite installations organize shared config into SiteAccess groups,
    # for single-site you can place shared config under "default"
```

A suggestion to make the tip easier to read